amespahdbpythonsuite/mcfitted.py

```diff
@@ -52,16 +52,22 @@ def __set(self, d: Optional[dict] = None, **keywords) -> None:
     def get(self) -> dict:
         """
         Calls class: :class:`amespahdbpythonsuite.transitions.Transitions.get`.
-        Assigns class variables from inherited dictionary.
-
+        Returns a dictionary including PAH UIDs and their corresponding weights.
         """
-        d = {}
-        d["type"] = self.__class__.__name__
-        d["mcfits"] = self.mcfits
-        d["distribution"] = self.distribution
-        d["observation"] = self.observation
 
-        return d
+        return {
+            "type": self.__class__.__name__,
+            "mcfits": [
```

Comment:
Originally "mcfits" held a list of Fiitted objects, allowing one to access all information wrt an individual fit. This functionality is now lost. I addition, one could obtain the uids and weights from each individual fit using the following code:

```python
fits = mcfit.get()
uids = [fit.getuids() for fit in fits["mcfits"]]
weights =[fit.getweights() for fit in fits["mcfits"]]
```